Makefile

```
@@ -1,34 +1,48 @@
-test: crt0.o
-	$(DMD) -betterC -c -version=$(VERSION) tests/tests.d -op -debug -m64
-	#$(DMD) -betterC -c -version=$(VERSION) tests/debug_print.d -op -debug -m64
-	ld -o tests.exe -static crt0.o stdio/printf.o string.o prng/rand.o stdarg.o exit/assert.o string/memcpy.o object.o tests/*.o #-macosx_version_min 10.7.0
+test: default crt0.o
+	$(DMD) -betterC -c -version=$(VERSION) tests/tests.d -op -debug $(DMD_FLAGS)
+	$(LD) -o tests.exe -static crt0.o stdio/printf.o string.o prng/rand.o stdarg.o exit/assert.o string/memcpy.o object.o posix/sys/writev.o tests/*.o #-macosx_version_min 10.7.0
 	./tests.exe
 
 all: libbutterc.so
 
 clean:
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)